### PR TITLE
Fix profiler

### PR DESCRIPTION
**Pull request by HaronK**

### CODE REVIEW

**Review by Flatfingers — APPROVED** [review]
I can't test this directly without a new executable, but I've reviewed the changes and they look reasonable.

**Review by Flatfingers — APPROVED** [review]

File: engine/lib/phx/src/system/profiler.rs [hunk 1]
```diff
@@ -16,8 +17,8 @@ pub type Signal = i32;
 pub struct Scope {
     pub name: *const libc::c_char,
     pub last: TimeStamp,
-    pub frame: u64,
-    pub total: u64,
+    pub frame: f64,
```
Comment (IllustrisJack) [inline engine/lib/phx/src/system/profiler.rs:20]: do we really need a float here?

Comment (HaronK) [reply engine/lib/phx/src/system/profiler.rs:20]: Yes, it's a number of seconds with fractional milliseconds

Comment (IllustrisJack) [reply engine/lib/phx/src/system/profiler.rs:20]: ah yeah it´s the delta time, for some reason thought this would just be a frame counter XD

**Review by IllustrisJack — APPROVED** [review]

**Review by dgavedissian — APPROVED** [review]

**Pull request merged**